packages/core-plugin-api/src/routing/RouteRef.ts

```diff
@@ -104,20 +65,13 @@ export function createRouteRef<
   id?: string;
```

Comment:
The `id` here should now be required, the previous logic was that you had to provide either `title` or `id`, now with the removal one has to provide an `id`